doc/release/1.5.0-notes.rst

```diff
@@ -80,21 +289,170 @@ Backwards incompatible changes
 
 `scipy.linalg` changes
 ----------------------
+The output signatures of ``?syevr``, ``?heevr`` have been changed from 
+``w, v, info`` to ``w, v, m, isuppz, info``
+
+The order of output arguments ``w``, ``v`` of ``<sy/he>{gv, gvd, gvx}`` is 
+swapped.
```

Comment:
Noting here for further review: check if these give hard errors; sounds like silently changing numerical values now.

Reply:
Only the argument order is changed to match the remaining ones. So I don't know if we can raise errors based on that or how.

Reply:
They do have different shape at least:
```
w : rank-1 array('d') with bounds (n)
v : rank-2 array('d') with bounds (n,n) and a storage
```

Still, it sounds like something to avoid. Was there a reason why it had to be done (e.g. infrastructure for a whole family of functions), or was it simply "let's make it consistent"?

Reply:
Probably they were hardly ever used. Because their lwork argument was not correct. So both fixed and made consistent.

Reply:
Okay that makes sense - I can try to add a sentence so that that bit doesn't read like people should go audit their code because we did something unusual.